java/cuvs-java/src/main/java22/com/nvidia/cuvs/internal/common/Util.java

```diff
@@ -247,8 +245,7 @@ public static BitSet concatenate(BitSet[] arr, int maxSizeOfEachBitSet) {
    * @return DLManagedTensor
    */
   public static MemorySegment prepareTensor(Arena arena, MemorySegment data, long[] shape, int code, int bits, int ndim,
-      int deviceType, int lanes) {
-
+      int deviceType, int lanes, MemorySegment strides) {
```

Comment:
Why do we need the strides passed in? All the calls to this seems to be MemorySegment.NULL at the moment.

Also, is this a remnant from some other work? Or is it related to the enums cleanup?